### PR TITLE
Validate $PONK

### PR DESCRIPTION
**Pull request by ponkcoin**
Ponk update

# Validate [{PONK}](https://solscan.io/token/HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support:https://x.com/Ponkcoin/status/1727258168710058231?s=20
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/ponk

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Review by GoncaloAS — COMMENTED** [review]
Great project

**Comment by ponkcoin** [comment]
> Great project

Thank you <3

File: validated-tokens.csv [hunk 1]
```diff
@@ -808,3 +808,4 @@ Pyth Network,PYTH,HZ1JovNiVvGrGNiiYvEozEVgZ58xaU3RKwX8eACQBCt3,6,https://pyth.ne
 FOMO,FOMO,Cx9oLynYgC3RrgXzin7U417hNY9D6YB1eMGw4ZMbWJgw,9,https://bt2uza2xywhm2tm5kjhm35xlxhtcqb3le272fir7rpnijpfbsnlq.arweave.net/DPVMg1fFjs1NnVJOzfbrueYoB2smv6KiP4vahLyhk1c,true
 Myro,$MYRO,HhJpBhRRn4g56VsyLuT8DL5Bv31HkXqsrahTTUCZeZg4,9,https://imageupload.io/ib/JgSESETVx7v60lk_1698972645.png,true
 Anatoly Coin,TOLY,to1yVXiNRMVVgS8i54Yjj3xB51MTorFrCMz7N8cirbK,9,https://shdw-drive.genesysgo.net/3YZtwGNZZisAUxKokcivJEwq2ag93APVXYqeyfaqb9Rq/toly-img.png,true
+Ponk,PONK,HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV,https://solscan.io/token/HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV,https://i.ibb.co/qg3VQVF/PONK-1.png,true
```
Comment (Timoon21) [inline validated-tokens.csv:811]:
```suggestion
Ponk,PONK,HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV,5,https://bafkreieyllpkega3ytsmq5gdwx2p44roka5pnlclw2an4xivqmbq7prmge.ipfs.nftstorage.link/,true
```

**Review by Timoon21 — DISMISSED** [review]

**Comment by bojes22216** [comment]
Good

**Review by benlvb — APPROVED** [review]

**Pull request merged**